ControlRoom/Controllers/SimCtl+SubCommands.swift

```diff
@@ -0,0 +1,745 @@
+//
+//  SimCtl+SubCommands.swift
+//  ControlRoom
+//
+//  Created by Patrick Luddy on 2/16/20.
+//  Copyright © 2020 Paul Hudson. All rights reserved.
+//
+
+import Foundation
+
+// swiftlint:disable file_length
+extension SimCtl {
+    struct Command {
+        let arguments: [String]
+
+        private init(_ subcommand: SubCommand, arguments: [String]) {
+            self.arguments = [subcommand.rawValue] + arguments
+        }
+
+        /// Create a new device.
+        static func create(name: String, deviceTypeId: String, runtimeId: String? = nil) -> Command {
+            Command(.create, arguments: [name, deviceTypeId])
+        }
+
+        /// Clone an existing device.
+        static func clone(deviceId: String, name: String) -> Command {
+            Command(.clone, arguments: [deviceId, name])
+        }
+
+        /// Upgrade a device to a newer runtime.
+        static func upgrade(deviceId: String, runtimeId: String) -> Command {
+            Command(.upgrade, arguments: [deviceId, runtimeId])
+        }
+
+        /// Delete spcified devices, unavailable devices, or all devices.
+        static func delete(_ delete: Delete) -> Command {
+            Command(.delete, arguments: delete.arguments)
+        }
+
+        /// Create a new watch and phone pair.
+        static func pair(watch: String, phone: String) -> Command {
+            Command(.pair, arguments: [watch, phone])
+        }
+
+        /// Unpair a watch and phone pair.
+        static func unpair(pairId: String) -> Command {
+            Command(.unpair, arguments: [pairId])
+        }
+
+        /// Set a given pair as active.
+        static func pairActivate(pairId: String) -> Command {
+            Command(.pairActivate, arguments: [pairId])
+        }
+
+        /// Erase a device's contents and settings.
+        static func erase(_ erase: Erase) -> Command {
+            Command(.erase, arguments: erase.arguments)
+        }
+
+        /// Boot a device.
+        static func boot(deviceId: String) -> Command {
+            Command(.boot, arguments: [deviceId])
+        }
+
+        /// Shutdown a device.
+        static func shutdown(_ shutdown: ShutDown) -> Command {
+            Command(.shutdown, arguments: shutdown.arguments)
+        }
+
+        /// Rename a device.
+        static func rename(deviceId: String, name: String) -> Command {
+            Command(.rename, arguments: [deviceId, name])
+        }
+
+        /// Print an environment variable from a running device.
+        static func getenv(deviceId: String, variable: String) -> Command {
```

Comment:
maybe `getEnvironmentValue`?

Reply:
how about `getEnvironmentVariable`? that way it matches the docs?